uaclient/cli.py

```diff
@@ -1558,6 +1564,10 @@ def action_attach(args, *, cfg):
                 event.info(msg.msg, file_type=sys.stderr)
                 event.error(error_msg=msg.msg, error_code=msg.name)
                 ret = 1
+
+        contract_client = contract.UAContractClient(cfg)
```

Comment:
We added this because the attach request itself can't include any services that got enabled. At spec time we decided we wanted data of what services got enabled immediately after attach (this could include `enableByDefault: true` services, or services that the user configured the pro-client to immediately enable). To get that data we decided to immediately send an activity ping after the post-attach-service-enables to get that info recorded.